src/main/java/randoop/main/GenTests.java

```diff
@@ -453,10 +522,55 @@ public boolean handle(String[] args) throws RandoopTextuiException {
     if (!GenInputsAbstract.noprogressdisplay) {
       System.out.printf("%nInvalid tests generated: %d", explorer.invalidSequenceCount);
     }
+    // TODO: output to file for sequence comparison between DigDog/Randoop
+    if (GenInputsAbstract.output_sequence_info) {
+      DigDogGenerator fExplorer =
+          (DigDogGenerator) explorer; // should work, explorer should always be a forw.gen.
```

Comment:
Comment seems extraneous -- a programmer wouldn't write code that isn't intended to work.  I would cut the comment.